.github/workflows/nightly-backward-compatibility.yml

```diff
@@ -18,17 +19,42 @@ on:
         default: "10.1.1"
 
 jobs:
+  prepare-env-variables:
+    runs-on: ubuntu-22.04
+    outputs:
+      total_releases: ${{ steps.set-env.outputs.total_releases }}
+      cardano_node_version: ${{ steps.set-env.outputs.cardano_node_version }}
+    steps:
+      - name: Prepare env variables
+        id: set-env
+        run: |
+          if [[ "${{ github.event_name }}" == "schedule" ]]; then
+            echo "total_releases=3" >> $GITHUB_OUTPUT
+            echo 'cardano_node_version=["10.1.1"]' >> $GITHUB_OUTPUT
```

Comment:
```suggestion
            echo 'cardano_node_version=["10.1.2"]' >> $GITHUB_OUTPUT
```

Reply:
Updated in this [commit](https://github.com/input-output-hk/mithril/pull/2083/commits/f6e0eb72fc860c9032d3198b18386ee6b6968003) ✅ .